.gitignore

```diff
@@ -18,7 +18,10 @@
 # Ignore code coverage folder
 coverage/*
 
+config/docker/Dockerfile.*
```

Comment:
No, it's just that I prepared some variants of Dockerfile to test some pieces of the stack. May I remove that entry?